ddtrace/tracer/option_test.go

```diff
@@ -249,6 +249,94 @@ func TestServiceName(t *testing.T) {
 	})
 }
 
+func TestTagSeparators(t *testing.T) {
+	assert := assert.New(t)
+
+	t.Run("env-tags", func(t *testing.T) {
+		for ind, tag := range []struct {
+			in       string
+			expected map[string]string
```

Comment:
```suggestion
			out map[string]string
```
Best friend of `in` is `out` 🙂